test/correctness/compute_at_update_stage.cpp

```diff
@@ -0,0 +1,37 @@
+#include <Halide.h>
```

Comment:
Actually this test name is too generic too, and there are no comments inside to explain what is being tested. Maybe rename the file to compute_at_reordered_update_stage and reference the original github issue by URL in a comment.

Reply:
Ok, will do